plugins/deye_plugin_ha-autoconfig.py

```diff
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+from deye_config import DeyeConfig
+from deye_mqtt import DeyeMqttClient, DeyeMqttPublishError
+from deye_plugin_loader import DeyePluginContext
+from deye_events import DeyeEventProcessor, DeyeEvent, DeyeObservationEvent
+
+from deye_sensor import SingleRegisterSensor
+from deye_sensors import sensor_list
+
+    
+class MqttAutoConfPublisher(DeyeMqttClient):
```

Comment:
Why is it necessary to extend `DeyeMqttClient` class?

Reply:
for publishing the config `publish_sensor`